### DISCUSSION TITLE
test Sewan ????

### DISCUSSION THREAD

**Discussion by xorgxx** [alexjustesen/speedtest-tracker#1719] [Q&A]
Hi it since that if i look in data base history, i can identify Sewan server always give low speed. even all statistics low are from Sewan.
so i have question it's the reference for m'y speed or not i mean how give the good stats ??

![2024-09-21_14-54-53](https://github.com/user-attachments/assets/d1dadca6-86c4-4392-b1a5-ce714a00d598)

my gest is that is not. so now how we can exclude same server ?

**Comment by svenvg93** [comment]
I don’t really understand the question, but if one server give low results all the time, it’s most likely a server limitation. The best solution would be to set the SPEEDTEST_SERVERS environment with the server id’s you want be used for the tests.

**Comment by xorgxx** [comment]
ok i will try this Thank.

**Comment by xorgxx** [comment]
ok i will try this Thank.
